gtda/diagrams/features.py

```diff
@@ -609,7 +609,8 @@ class PersistenceImage(BaseEstimator, TransformerMixin):
 
     _hyperparameters = {'sigma': [numbers.Number, (1e-16, np.inf)],
                         'n_bins': [int, (1, np.inf)],
-                        'effective_weight_function_': [types.FunctionType]}
+                        'effective_weight_function_': [types.FunctionType,
```

Comment:
Looking at `fit`, `effective_weight_function_` by construction can never be `None`. What's the reason for this change and for the analogous changes below?

Reply:
No, `None` here just. mean that this hyperparameter value does is not constrained.

Reply:
Thanks for clarifying!

Reply:
@wreise perhaps you'd also like to know this.